frontend/src/scenes/pipeline/Overview.tsx

```diff
@@ -0,0 +1,273 @@
+import { LemonCard, LemonSkeleton, LemonTag, Link, Tooltip } from '@posthog/lemon-ui'
+import clsx from 'clsx'
+import { useValues } from 'kea'
+import { SeriesGlyph } from 'lib/components/SeriesGlyph'
+import { More } from 'lib/lemon-ui/LemonButton/More'
+import { LemonMarkdown } from 'lib/lemon-ui/LemonMarkdown'
+import { percentage } from 'lib/utils'
+import { urls } from 'scenes/urls'
+
+import { PipelineAppKind, PipelineAppTab, PluginConfigWithPluginInfoNew } from '~/types'
+
+import { AppMetricSparkLine } from './AppMetricSparkLine'
+import { DestinationMoreOverlay } from './Destinations'
+import { DestinationType, PipelineAppBackend } from './destinationsLogic'
+import { pipelineOverviewLogic } from './overviewLogic'
+import { pipelineAppMetricsLogic } from './pipelineAppMetricsLogic'
+import { TransformationsMoreOverlay } from './Transformations'
+import { pipelineTransformationsLogic } from './transformationsLogic'
+import { humanFriendlyFrequencyName } from './utils'
+
+const FAILURE_RATE_WARNING_THRESHOLD = 0
+const FAILURE_RATE_ERROR_THRESHOLD = 0.03
+
+type StatusMetrics = {
+    totals: number
+    failures: number
+}
+
+type StatusIndicatorProps = {
+    enabled: boolean
+    metrics?: StatusMetrics
+}
+
+const StatusMessage = ({ enabled, metrics }: StatusIndicatorProps): JSX.Element => {
+    if (!enabled) {
+        return <i>Disabled.</i>
+    }
+
+    if (!metrics) {
+        return (
+            <span>
+                Enabled - <i>No events processed in the last 7 days.</i>
+            </span>
+        )
+    }
+
+    const failureRate = metrics.failures / metrics.totals
+
+    if (metrics.failures > 0) {
+        return (
+            <span>
+                {metrics.totals} events processed with {percentage(failureRate)} failures (a total of {metrics.failures}{' '}
+                event(s)) in the last 7 days.
+            </span>
+        )
+    }
+
+    return <span>{metrics.totals} events processed without errors in the last 7 days.</span>
+}
+
+const StatusIndicator = ({ enabled, metrics }: StatusIndicatorProps): JSX.Element => {
```

Comment:
Idea: let's remove the status indicator completely:
1. lets not show disabled apps at all (there could be lots of them and they aren't relevant in the moment)
2. the sparkLine gives us a great overview of how the app is doing already, having another colored dot could create more confusion than help 🤷‍♀️